### PR TITLE
longlong-double-fix resurrection part 1

### PR DESCRIPTION
**Pull request by mohawk2**

### CODE REVIEW

**Comment by mohawk2** [comment]
Temporary close because Travis fouls up with race condition when both branch and PR.

File: Basic/Core/Types.pm.PL [hunk 1]
```diff
@@ -143,6 +143,15 @@ my @types = (
 		  packtype => 'd*',
 	          defaultbadval => '-DBL_MAX',
 	      },
+	      {
+		  identifier => 'A',
+		  pdlctype => 'PDL_Anyval',
+		  realctype => 'double',
+		  ppforcetype => 'anyval',
+		  usenan => 0, # chm set this to 1, but then get compile error
```
Comment (zmughal) [inline Basic/Core/Types.pm.PL]: This comment needs to be more descriptive or at least the underlying issue should be looked at.

**Comment by zmughal** [comment]
The changes in this look good to me. And it seems to increase the code coverage.

**Comment by zmughal** [comment]
It looks good to me. Just waiting on Travis.

**Pull request merged**